content/news/2023-10-21-bevy-0.12/index.md

```diff
@@ -15,9 +15,13 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 <!-- more -->
 
-## Feature Name
+## Wireframe Improvements
 
-<div class="release-feature-authors">authors: @author</div>
+<div class="release-feature-authors">authors: @IceSentry</div>
+
+The wireframes now use bevy's Material abstraction. This means it will automatically use the new batching and instancing features while being easier to maintain. This change also made it easier to add support for colored wireframe. You can configure the color globally or per mesh using the `WireframeColor` component. It's also now possible to disable wireframe rendering by using the `NeverRenderWireframe` component.
+
+![wireframe](wireframe.png)
```

Comment:
I think this example would visually benefit from using slightly less max-RGB coloring for the base colors (and maybe slightly brighter lighting). Ex: something closer to the `3d_scene` palette.

Reply:
Yeah, good idea. I just used the default wireframe example. I'll update it with nicer colors

Reply:
How about this?

I used uvspheres and more divisions on the plane to show more wireframe on screen

![image](https://github.com/bevyengine/bevy-website/assets/8348954/49a2f6f9-9824-42e1-9a9b-ad227079a67f)

Reply:
Maybe with a slightly less dense sphere mesh

![image](https://github.com/bevyengine/bevy-website/assets/8348954/dd4c505c-2616-40d6-9d50-6d24347ff340)

Reply:
I've pushed that last one. Let me know if you would like any other changes

Reply:
Sorry for the late reply. Love the changes!